ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Storage/ChainDB/API.hs

```diff
@@ -887,7 +887,7 @@ data LoE a =
   -- When the selection's tip is @k@ blocks after the earliest intersection of
   -- of all candidate fragments, ChainSel will not add new blocks to the
   -- selection.
-  LoEEnabled a
+  LoEEnabled !a
```

Comment:
Done in https://github.com/IntersectMBO/ouroboros-consensus/pull/1113/commits/dd651b114624cbe86486805a256387f8adbdfb45